src/v1.js

```diff
@@ -759,7 +759,12 @@ api.declare({
     "the secret credentials leave this service."
   ].join('\n')
 }, function(req, res) {
-  return this.signatureValidator(req.body).then(result => res.reply(result));
+  return this.signatureValidator(req.body).then(result => {
+    if (result.expires) {
```

Comment:
the property is required... at least according to the schema... but from what I can see it's always there.